src/components/Datepicker/Datepicker.js

```diff
@@ -120,24 +220,33 @@ export class Datepicker extends React.Component {
     );
   }
 
+  /**
+   * Renders the monthly view including weeks and days
+   * @return {ReactElement} tbody
+   */
   renderMonth() {
+    const today = this.props.translations.today;
```

Comment:
This can easily break since  `today` and `translations` are not required. Consider requiring the props ore use a safe getter, e.g. lodashs' `get(this.props, 'translations.today', 'Today')` with a default value.